pyabc/visualization/data.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import logging
+
+
+logger = logging.getLogger("Data_plot")
+
+
+def plot_data(obs_data: dict, sim_data: dict, key=None):
+    """
+    Plot summary statistic data.
+
+    Parameters
+    ----------
+
+    obs_data: dict
+        A dictionary for the summary statistic of the observed data,
+        where keys represent the summary statistic name and values represent
+        the data itself. The values can be represented as pandas dataframe,
```

Comment:
here in the function help says "represented as pandas dataframe" not dictionary. Please check consistency

Reply:
yes, it should always be pandas dataframe, entries cannot be dicts.